modules/xfem/src/base/XFEM.C

```diff
@@ -923,11 +924,14 @@ XFEM::healMesh()
   std::set<Node *> nodes_to_delete;
   std::set<Node *> nodes_to_delete_displaced;
   std::set<unsigned int> cutelems_to_delete;
+  unsigned int deleted_elem_count = 0;
+  std::vector<std::string> healed_geometric_cuts;
 
   for (unsigned int i = 0; i < _geometric_cuts.size(); ++i)
   {
     if (_geometric_cuts[i]->shouldHealMesh())
     {
+      healed_geometric_cuts.push_back(_geometric_cuts[i]->name());
```

Comment:
shouldn't you only push onto this vector id `_debug_output_level > 0` ?

Reply:
I'd be inclined to do it anyway just because it's not terribly expensive to do (once per time step, and usually a small handful of strings), and someone might want to use it for something else down the road, and might wonder why it's empty.

Reply:
Yeah ok, once per timestep is not an issue.